src/ranged.cpp

```diff
@@ -155,9 +155,8 @@ static const std::set<material_id> ferric = { material_iron, material_steel, mat
 static constexpr int AIF_DURATION_LIMIT = 10;
 
 static projectile make_gun_projectile( const item &gun );
-static int NPC_time_to_attack( const Character &p, const itype &firing );
-static int time_to_attack( const Character &p, const item &firing, const item_location &loc );
-static int RAS_time = 0;
+static int time_to_attack( const Character &p, const itype &firing );
+static int RAS_time( const Character &p, const item &firing, const item_location &loc );
```

Comment:
I rewrote this part in a3eb1559a62ac8c53a7a71f09af3e05d182cfe3c, can you review it again? @kevingranade